client/src/main/proto/spine/validation.proto

```diff
@@ -61,19 +61,23 @@ extend google.protobuf.FieldOptions {
 
 // A validation constraint violation for a message field.
 message ConstraintViolation {
-    // An interpolated error message for this constraint violation.
+    // An error message for this constraint violation
+    // (or a template which contains `%s` format specifiers).
     string message = 1;
 
-    // A non-interpolated error message for this constraint violation.
-    string message_template = 2;
+    // Parameters (as strings) referenced by the format specifiers in the error message format string.
```

Comment:
1. `Value` doesn't have `long` kind field, only `double`. And the `long` value can be rounded (if it is greater than max double value). 
2. It is more convenient to use `string` params for message formatting. I think they won`t be used in other ways. Also, we pass the min/max value to the options as strings, too (to support both `long` and `double` types).